iree/testing/benchmark_main.c

```diff
@@ -16,8 +16,26 @@
 #include "iree/testing/benchmark.h"
 
 int main(int argc, char** argv) {
+  // Pass through flags to benchmark.
+  // Note that we handle --help so we have to include benchmark's flags here.
```

Comment:
:-( If we're writing our own flag library, can we try to make it play nicer with other flag parsing? Like handling `--help` but allowing parsing to continue? I think I could probably even send a patch to benchmark to factor it such that it has a method that can do the same.

Reply:
I didn't want to change too much related to that here until I had a chance to port our other benchmark usage to the `iree/testing/benchmark.h` API. Benchmark not exiting on --help would be fantastic.

Reply:
Filed #5628 to improve our side.

Reply:
Writing up the issue was enough to make the solution easy. Done :)

Reply:
:tada: thanks!